contents/builder-community-incubator.mdx

```diff
@@ -0,0 +1,140 @@
+  </p>
+  
+</div>
+
+<div style={{ display: 'flex', justifyContent: 'center', margin: '2rem 0 2.5rem 0' }}>
+  <img
+    src="https://res.cloudinary.com/dmukukwp6/image/upload/f_webp,h_680,w_1209,c_fill,g_auto,q_auto/mykonoshackathon_979cf7bec6?_a=AXAH4nI0"
+    alt="Builders at Mykonos Hackathon"
+    style={{ width: '75%', maxWidth: '600px', borderRadius: '1rem', boxShadow: '0 4px 24px rgba(0,0,0,0.08)' }}
+  />
+</div>
+
+<div className="my-24 px-5">
+  <div className="max-w-screen-2xl mx-auto">
+    <h3 className="text-2xl lg:text-4xl m-0 text-center mb-6 sm:mb-16 lg:mb-1">Here's how we support organizers</h3>
+    <div className="flex justify-center">
+      <ul className="m-0 p-0 list-none inline-grid sm:grid-cols-2 lg:grid-cols-3 gap-[10px] justify-evenly relative text-center sm:text-left">
+        <li className="bg-[#E5E7E0] dark:bg-[#2C2C2C] relative md:max-w-md py-4 md:py-8 pl-4 pr-2 md:px-12 lg:px-8 rounded-2xl shadow-sm transition-colors duration-300">
+          <h5 className="text-xl font-extrabold m-0 pb-1 pr-4 text-black dark:text-white">$800 cash grant</h5>
+          <p className="m-0 text-[15px] text-gray-800 dark:text-gray-300">
+            Meant to get you going and to be used at organizer's discretion.
+          </p>
+        </li>
+        <li className="bg-[#E5E7E0] dark:bg-[#2C2C2C] relative md:max-w-md py-4 md:py-8 pl-4 pr-2 md:px-12 lg:px-8 rounded-2xl shadow-sm transition-colors duration-300">
+          <h5 className="text-xl font-extrabold m-0 pb-1 pr-4 text-black dark:text-white">Help attracting attendees</h5>
+          <p className="m-0 text-[15px] text-gray-800 dark:text-gray-300">
+            We'll spread the word on your behalf if you need it.
+          </p>
+        </li>
+        <li className="bg-[#E5E7E0] dark:bg-[#2C2C2C] relative md:max-w-md py-4 md:py-8 pl-4 pr-2 md:px-12 lg:px-8 rounded-2xl shadow-sm transition-colors duration-300">
+          <h5 className="text-xl font-extrabold m-0 pb-1 pr-4 text-black dark:text-white">Help finding a venue</h5>
+          <p className="m-0 text-[15px] text-gray-800 dark:text-gray-300">
+            The ideal location is a free space which we can source.
+          </p>
+        </li>
+      </ul>
+    </div>
+  </div>
+</div>
+
+<div className="grid md:grid-cols-2 gap-8 items-start my-12">
+  <div>
+    <h2 className="text-3xl lg:text-4xl mb-6">What we're asking from you, pilot participants</h2>
+    <p>
+      Bring together a group of at least 10 people to co-work and build for 5 straight weeks. The goal is to give you the initial push but let you carry forward afterwards.
+    </p>
+    <p>
+      Create a consistent time and structure that works for your group. For times, do evenings or weekends. For format, we suggest two ingredients: building and demo-ing.
+    </p>
+    <p>
+      Let us know how it's going along the way.
```

Comment:
Could be great to offer something here? Like, 5 weeks is a big commitment. We should try to be clear about the incentive. Soft feeling though.